actions/llm-pr-writer/action.yml

```diff
@@ -0,0 +1,138 @@
+name: llm-pr-writer
+description:
+  "This GitHub workflow harnesses the capabilities of Large Language Models
+  (LLMs) to automate the creation and enhancement of pull request (PR)
+  descriptions. Designed to streamline the development process, it significantly
+  reduces the time and effort traditionally required to draft comprehensive PR
+  descriptions, particularly for extensive changes. By meticulously analyzing
+  the modifications within a PR, the workflow generates precise, context-rich
+  summaries that facilitate improved collaboration and review efficiency."
+
+inputs:
+  # general inputs
+  checkout-repo:
+    description: "enable git checkout repo"
+    required: false
+    default: "true"
+  checkout-repo-fetch-depth:
+    description: "number of commits to fetch"
+    required: false
+    default: "0"
```

Comment:
In what scenario would people want to checkout vs not checkout?

Reply:
good point